cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -150,8 +150,9 @@ def _validate(self, custom_ami: str):
         elif PCLUSTER_IMAGE_BUILD_STATUS_TAG not in tags_dict:
             self._add_failure(
                 (
-                    "The custom AMI did not pass the tests in image builder. "
+                    "Unable to retrieve image build status."
```

Comment:
what about 
"Unable to retrieve custom AMI build status. Please check build-image CloudFormation stack for details."

Reply:
Done